src/main/java/domain/AbsencePenalty.java

```diff
@@ -0,0 +1,35 @@
+package domain;
+
+public enum AbsencePenalty {
+    OUT("제적", 5),
+    MEETING("면담", 3),
+    WARNING("경고", 2),
+    NORMAL("정상", 0);
+
+    public static final int LATE_TO_ABSENCE_THRESHOLD = 3;
+    private final String level;
+    private final int standard;
+
+    AbsencePenalty(String level, int standard) {
+        this.level = level;
+        this.standard = standard;
+    }
+
+    public static AbsencePenalty findAbsenceLevel(int absentCount, int lateCount) {
+        absentCount += (lateCount / LATE_TO_ABSENCE_THRESHOLD);
```

Comment:
받은 메서드 파라미터를 수정하기보다는 필요하다면 별도의 변수를 둬서 사용하는 것이 좋다고생각해요. 

```suggestion
    public static AbsencePenalty findAbsenceLevel(final int absentCount, final int lateCount) {
        int calculatedAbsentCount = absentCount + (lateCount / LATE_TO_ABSENCE_THRESHOLD);
```

Reply:
좋은 의견 감사합니다!😊